features/TESTS/filesystem/heap_block_device/main.cpp

```diff
@@ -78,30 +74,15 @@ void test_read_write() {
             write_block[i] = 0xff & rand();
         }
 
-        // erase, program, and read the block
-        printf("test  %0*llx:%llu...\n", addrwidth, block, block_size);
-
         err = bd.erase(block, block_size);
         TEST_ASSERT_EQUAL(0, err);
 
         err = bd.program(write_block, block, block_size);
         TEST_ASSERT_EQUAL(0, err);
 
-        printf("write %0*llx:%llu ", addrwidth, block, block_size);
-        for (int i = 0; i < 16; i++) {
-            printf("%02x", write_block[i]);
-        }
-        printf("...\n");
-
         err = bd.read(read_block, block, block_size);
         TEST_ASSERT_EQUAL(0, err);
 
-        printf("read  %0*llx:%llu ", addrwidth, block, block_size);
-        for (int i = 0; i < 16; i++) {
-            printf("%02x", read_block[i]);
-        }
-        printf("...\n");
```

Comment:
Let me see if I can revert that, as I manage to shave some memory somewhere else, it may not be necessary. On the side note, this sort of prints in test shouldn't be necessary. We should be asserting things we care about in a way that when they fail we will know what failed from assert not from a print.